### PR TITLE
(GH-325) Update Cake to 2.0

### PR DESCRIPTION
**Pull request by twenzel**
- Update Cake reference to 2.0
- Update packages
- Increase version to 1.2.0

### CODE REVIEW

**Comment by twenzel** [comment]
@pascalberger Please have a look.

**Comment by twenzel** [comment]
@augustoproiete are you also a maintainer of this repository?

**Comment by pascalberger** [comment]
> @pascalberger Please have a look.

@twenzel Thanks for your contribution 👍 

I already had a look at the PR. Unfortunately I can't merge it like this, since it mixes different things in a single PR (update of build, targeting a new version of Cake, style changes). For a clean history I prefer separate pull requests for separate concerns. 

Can you please split the PR for separate concerns. Also happy to discuss changes in advance in an issue to avoid you doing work which won't be merged afterwards.

**Comment by augustoproiete** [comment]
@twenzel I'm not, but @pascalberger got you covered

File: src/Cake.AzureDevOps/Cake.AzureDevOps.csproj
Comment (pascalberger) [inline src/Cake.AzureDevOps/Cake.AzureDevOps.csproj:25]: This is not required in this PR (will become version 2.0 later)

File: src/Cake.AzureDevOps.Tests/Cake.AzureDevOps.Tests.csproj
Comment (pascalberger) [inline src/Cake.AzureDevOps.Tests/Cake.AzureDevOps.Tests.csproj]: Please only update required packages. Dependabot will take care of the other later

**Review by pascalberger — CHANGES_REQUESTED** [review]

**Review by pascalberger — APPROVED** [review]
LGTM

**Comment by pascalberger** [comment]
@twenzel your changes have been merged, thanks for your contribution 👍

**Pull request merged**
